.github/workflows/core-4-label-reminder.yml

```diff
@@ -0,0 +1,33 @@
+name: Core 4 Label Reminder
+
+# Run when a PR is opened or marked ready for review, but not for drafts (see
+# condition in job below)
+on:
+  pull_request:
+    types:
+      - opened
+      - ready_for_review
+
+permissions:
+  pull-requests: write
+
+jobs:
+  core_4_label_reminder:
+    if: >-
+      github.event.pull_request.draft == false &&
+      contains(github.event.pull_request.labels.*.name, 'maintenance') == false &&
+      contains(github.event.pull_request.labels.*.name, 'feature') == false &&
+      contains(github.event.pull_request.labels.*.name, 'fix') == false
```

Comment:
Will this match any label with 'fix' in the name, not just a label which is exactly named 'fix'?

Reply:
Oh good question. I think the `contains` checks whether `fix` is present in the list of labels. I'll test it out.

Reply:
I think this is behaving as expected. I added a label `fixxxx` and then marked this PR as ready for review and still got the comment:

<img width="870" height="338" alt="Screenshot 2025-10-23 at 10 28 43" src="https://github.com/user-attachments/assets/b82b5da9-9518-4651-bbfe-0b07a9333bb0" />